src/main/scala/graphql/schema/CandidateModeType.scala

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package basic.graphql.schema
+
+import basic.ObservingMode
+import basic.search.Constraints
+import cats.effect._
+import sangria.schema._
+
+object CandidateModeType {
+
+  def apply[F[_]: Effect]: UnionType[Unit] =
+    UnionType(
+      name        = "CandidateModeType",
+      description = Some("Candidate observing mode."),
+      types       = List(spectroscopy[F])
+    )
+
+  def spectroscopy[F[_]: Effect]: ObjectType[Unit, (Constraints.Spectroscopy, ObservingMode.Spectroscopy)] =
```

Comment:
Everything is parameterized on an unused effect type `F` in anticipation that we'll need to start hitting the database or calling out to ITC at some point.